duet/views/mobile_android_country.view.lkml

```diff
@@ -0,0 +1,226 @@
+view: mobile_android_country {
+  derived_table: {
+    sql: with last_updated as (
```

Comment:
Swapping between apps is done through an app id parameter and seems to work okay. I'm not sure how this would work in the context of a bigquery-etl view because of all the different parameterizations (e.g. how big should the current aggregation window be, which tables should we actually be using to count first seen clients, etc).